mkdocs.yml

```diff
@@ -11,6 +11,7 @@ validation:
   omitted_files: warn
   absolute_links: warn
   unrecognized_links: warn
+  anchors: warn
```

Comment:
I think Renovate _just_ pulled in the new Git submodule for `renovatebot/renovate` that has your new relative links checks from (https://github.com/renovatebot/renovate/pull/28690):

- https://github.com/renovatebot/renovatebot.github.io/commit/8de72a886fe77171a1467526e72bcf40a75a180b

Please double check before merging. :wink: